mlflow/gateway/config.py

```diff
@@ -372,10 +374,14 @@ def validate_model(cls, model):
                 )
         return model
 
-    @root_validator(skip_on_failure=True)
```

Comment:
what about this logic? are we losing it?

Reply:
skip_on_failure is no longer available in Pydantic V2, unfortunately.

Reply:
Yeah but for pydantic v1 it still needs to work since it is supported

Reply:
Good point. Updated to address that comment 👍